### PR TITLE
Add rate limiter for rocksdb compaction

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
Add rate limiter for rocksdb compaction. Depend on this [pr](https://github.com/ngaut/rust-rocksdb/pull/57).

### CODE REVIEW

**Comment by zhangjinpeng1987** [comment]
ref https://github.com/pingcap/tikv/issues/1256

**Comment by zhangjinpeng1987** [comment]
@siddontang @BusyJay PTAL

**Comment by BusyJay** [comment]
LGTM

File: etc/config-template.toml [hunk 1]
```diff
@@ -155,6 +155,9 @@ stats-dump-period-sec = 600
 # least 2MB;
 # compaction_readahead_size = 0
 
+# Limit the disk IO of compaction and flush.
+# rate-bytes-per-sec = "100MB"
```
Comment (BusyJay) [inline etc/config-template.toml]: I don't think we should leave "100MB" here, it may make people think that's the default settings.

Comment (siddontang) [inline etc/config-template.toml]: +1

**Comment by zhangjinpeng1987** [comment]
@BusyJay PTAL

**Pull request merged**